api/configs/configs.py

```diff
@@ -0,0 +1,30 @@
+import importlib
+
+class Config(object):
+    HOST = 'localhost'
+    PORT = 8080
+    DEBUG = True
+    USE_RELOADER = True
+
+# add class for new configuration profile here
+# i.e. class ProductionConfig(Config) ...
+
+
+def compile_config(profile: str) -> Config:
+    return _compile_config(profile, 'configs.personal', 'PersonalConfig')
+
+# Separate function signature for testing.
+def _compile_config(profile: str, mod: str, clazz: str) -> Config:
+    config = Config()
+
+    if profile == 'personal':
+        try:
+            personal_mod = importlib.import_module(mod)
+            personal_config = getattr(personal_mod, clazz)
+            config = personal_config
+        except ModuleNotFoundError:
+            raise
```

Comment:
we halt if the 'personal' profile is set but theres no mod and class for personal configs